linkcheck/strformat.py

```diff
@@ -216,15 +225,21 @@ def strduration(duration):
     else:
         prefix = ""
     duration = math.ceil(duration)
-    if duration >= SECONDS_PER_HOUR: # 1 hour
+    if duration >= SECONDS_PER_HOUR:  # 1 hour
         # time, in hours:minutes:seconds
-        return "%s%02d:%02d:%02d" % (prefix, duration // SECONDS_PER_HOUR,
-                                   (duration % SECONDS_PER_HOUR) // SECONDS_PER_MINUTE,
-                                    duration % SECONDS_PER_MINUTE)
+        return "%s%02d:%02d:%02d" % (
+            prefix,
+            duration // SECONDS_PER_HOUR,
+            (duration % SECONDS_PER_HOUR) // SECONDS_PER_MINUTE,
+            duration % SECONDS_PER_MINUTE,
+        )
     else:
         # time, in minutes:seconds
-        return "%s%02d:%02d" % (prefix, duration // SECONDS_PER_MINUTE,
-                                 duration % SECONDS_PER_MINUTE)
+        return "%s%02d:%02d" % (
+            prefix,
+            duration // SECONDS_PER_MINUTE,
+            duration % SECONDS_PER_MINUTE,
+        )
 
 
 # from quodlibet
```

Comment:
whee, more vendored code. there's stdlib stuff for that as well... i guess another thing to cleanup i hadn't noticed...

Reply:
Included in  #433